scripts/install-theme.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+mv "$DESTDIR"/"$1" "$DESTDIR"/"$2"
```

Comment:
i don't like the idea with extracting it into separate shell script

Reply:
I took the idea from their official documentation https://mesonbuild.com/Installing.html#custom-install-behavior

I don't personally mind how it is solved as long as it is.